src/sentry/migrations/0517_backfill_pagerdutyservices_into_org_integrations.py

```diff
@@ -16,6 +16,11 @@ def as_dict(pds):
 def backfill_pagerdutyservices(apps, schema_editor):
     PagerDutyService = apps.get_model("sentry", "PagerDutyService")
     OrganizationIntegration = apps.get_model("sentry", "OrganizationIntegration")
+    try:
+        PagerDutyService.objects.first()
+    except ProgrammingError:
+        # Table was not created as the pagerdutyservice model has been removed.
```

Comment:
Why is this necessary? I would have thought the hint would be enough to redirect it to the correct database?